server/templates/pages/src/terms.html

```diff
@@ -35,11 +34,7 @@ <h1 id="fxa-tos-header">{{#t}}Terms of Service{{/t}}</h1>
           </div>
         </div>
 
-        <a id="about-mozilla" rel="author" target="_blank" href="https://www.mozilla.org/about/?utm_source=firefox-accounts&amp;utm_medium=Referral"></a>
-
-        <!--[if lt IE 10]>
-            <p class="browsehappy">{{#t}}You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.{{/t}}</p>
-        <![endif]-->
-
+        <a id="about-mozilla" rel="author" target="_blank" href="https://www.mozilla.org/about/?utm_source=firefox-accounts&amp;utm_medium=Referral">
+      </a>
```

Comment:
@MynahMarie Hey! could you please put the closing `</a>` on the same like it was before?